docs/olm.md

```diff
@@ -8,15 +8,27 @@ Ensure the Hive namespace exists:
 oc new-project hive
 ```
 
-Identify the current version of the operator you wish to replace, if applicable. (TODO)
+Identify the current version of the operator you wish to replace, if applicable. If not, you should be able to just specify a fake version.
 
 ```
 GIT_HASH=`git rev-parse --short HEAD`
 GIT_COMMIT_COUNT=`git rev-list 9c56c62c6d0180c27e1cc9cf195f4bbfd7a617dd..HEAD --count`
 ./hack/generate-operator-bundle.py bundle/ 0.1.5-4b53b492 $GIT_COMMIT_COUNT $GIT_HASH quay.io/dgoodwin/hive:latest alpha
 ```
 
-Build a registry image containing your bundle and publish to quay:
+You should now have a ./bundle/[version]/ directory.
+
+You now need to create a package file for the bundle in ./bundle/hive.package.yaml
+
+
+```
+packageName: hive-operator
+channels:
+- name: alpha
+  currentCSV: CURRENTCSVNAME
```

Comment:
It'd be nice if we have add a simple script that will create it for you

Reply:
Going down this path, I ended up just making the whole doc a script anyone who wants to try can work off.